Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableOnTouchReceiver.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// A basic touch event receiver for detecting Physical Touch state changes in the Interactable
+    /// When the physical touch states change, these events are triggered.
+    /// </summary>
+    public class InteractableOnTouchReceiver : ReceiverBase
+    {
+        [InspectorField(Type = InspectorField.FieldTypes.Event, Label = "On Touch Off", Tooltip = "Touch has left the object")]
+        public UnityEvent OnTouchOff = new UnityEvent();
```

Comment:
Why is there OnTouchOff but on OnTouchOn or OnTouchBegin?

Reply:
There is an OnTouch event. All ReceiverBase classes come with a Unity event which in this case is called OnTouch. OnTouchOff is the second event. Maybe OnTouchStart and OnTouchEnd would be better, or start/complete?